### DISCUSSION TITLE
Messages appear on both sides to clear confusion

### DISCUSSION THREAD

**Discussion by aliraad79** [element-hq/element-meta#1727] [Ideas]
Hi.
I'm using element for some times now and everything is good for me. just yesterday i install it on my grandmas phone and she had a lot of confusion because of messages for all of people in a chat appear in same side unlike some apps like Telegram or Whatsapp.
I wonder if there is a technical difficulty that you had not develop this and if there is not I be happy to work on it and at least make it an feature.

ps: Sorry if it's duplicated i try my best to find any topic but doesn't found one.

**Comment by aliraad79** [comment]
I forget to mention that we use android version.

**Reply by t3chguy** [reply]
You can turn on chat bubbles 

Settings > Preferences > Timeline > Show Message bubbles
